### DISCUSSION TITLE
Pls.Pls can this be put up for Unraid.

### DISCUSSION THREAD

**Discussion by ionultd** [JarvyJ/HomeIntent#218] [Ideas]
It would be awesome if this was available via Unraid apps.

Thanks

**Comment by JarvyJ** [comment]
Hello! I'm not entirely sure what that is, or what it would take to support it. Do I need specific hardware/software to make it happen? Also, right now since satellite support isn't in yet, you have to physically plug in the mic/speaker into the device.

Let me know, and we can maybe take a look at it down the road.

**Reply by n8ohu** [reply]
@JarvyJ UnRaid is a system similar in design to TrueNAS or other Network Attached Storage systems that has the ability to run Docker Containers or something similar.
